src/olympia/addons/management/commands/process_addons.py

```diff
@@ -31,8 +31,12 @@
     'sign_addons': {'method': sign_addons, 'qs': []},
     'add_firefox57_tag_to_webextensions': {
         'method': add_firefox57_tag,
-        'qs': [Q(status=amo.STATUS_PUBLIC,
-                 _current_version__files__is_webextension=True)]},
+        'qs': [
+            Q(status=amo.STATUS_PUBLIC) & (
+                Q(_current_version__files__is_webextension=True) |
+                Q(_current_version__files__is_mozilla_signed_extension=True)
```

Comment:
We could actually filter this by `_current_version__apps__max__version_int__gte=firefox_57_star` but I wonder if that's correct, hmm.

Reply:
it's fine like this IMHO

Reply:
mhmm, well, we should set `strict_compatibility` to whatever is set in their manifest for existing ones. I'll add a task for that on Monday, that's a very good point.